.buildkite/pipeline.yml

```diff
@@ -102,7 +102,16 @@ steps:
   #     ./result/bin/baseDAO-test --nettest-run-network
   #       --pattern '\$1 == "On network" || \$NF == "On network" || \$0 ~ /.On network./'
 
+  - label: check typescript api
+    depends_on:
+      - build-ligo
+      - build-haskell
+    commands:
+      - nix-build ci.nix -A packages.baseDAO-ligo-meta.tests.baseDAO-test
```

Comment:
AFAICT this step is unnecessary, as the `baseDAO-ligo-meta` package is already built by `build-haskell` and tests aren't necessary to check the typescript diff.

Reply:
Turned out to be true. Have removed it.